sponsorship.md

```diff
@@ -12,6 +12,18 @@ SoCraTes UK is a non-profit conference organised by and for the Software Crafter
 our attendees only need to pay for their hotel room. As such, we rely heavily on our sponsors to reduce 
 the overall cost of attendance and in exchange, we provide exposure (and well deserved Kudos!) to all of our sponsors.
 
+<br>
```

Comment:
Not sure those new lines are necessary?

Reply:
We have them on all pages. 
I had a look what it looks like without them when we did the bigger changes on the website and it becomes incredibly vertical dense. 

It is definitely easier to read when we have them in